src/lib/graphqlErrorHandler.ts

```diff
@@ -125,7 +120,7 @@ export const graphqlErrorHandler = (
     const flattenedErrors = flattenErrors(topLevelError)
     if (enableSentry) {
       flattenedErrors.forEach(e => {
-        if (shouldReportError(e) || shouldReportParentError(e.originalError)) {
```

Comment:
What happened was that for any error other than syntax errors, `shouldReportError` would return `true` and thus never perform the checks of `shouldReportParentError`. I now rolled that into one.

Reply:
ace